app/api/dao/user.py

```diff
@@ -58,18 +89,63 @@ def delete_user(user_id):
 
     @staticmethod
     def get_user(user_id):
+        """Finds a user by id.
+
+        Returns a user which has been searched by their id.
```

Comment:
This explanation is not needed. You can remove this because the first line explains everything ;)